src/components/jacobian_component/jacobian.sh

```diff
@@ -178,12 +213,23 @@ run_jacobian() {
         # inversion w/ precomputed Jacobian
         if "$KalmanMode"; then
             cd ${RunDirs}/kf_inversions/period${period_i}
-            precomputedJacobianCache=${ReferenceRunDir}/kf_inversions/period${period_i}/data_converted
+            precomputedJacobianCachePrefix=${ReferenceRunDir}/kf_inversions/period${period_i}
         else
             cd ${RunDirs}/inversion
-            precomputedJacobianCache=${ReferenceRunDir}/inversion/data_converted
+            precomputedJacobianCachePrefix=${ReferenceRunDir}/inversion
         fi
-        ln -s $precomputedJacobianCache data_converted_reference
+
+        precomputedJacobianCacheNormal=${precomputedJacobianCachePrefix}/data_converted
+        precomputedJacobianCacheLognormal=${precomputedJacobianCachePrefix}/data_converted_background
```

Comment:
is the `date_converted_background` directory where the Jacobian matrix elements are saved in the Lognormal errors case? If so, any reason why? Just checking.

Reply:
Same reason as above for the `data_geoschem_background` dir. Will work on merging them